build/monaco/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "monaco-editor-core",
   "private": true,
-  "version": "0.31.1-os",
+  "version": "0.31.1-os2-cnh-3",
```

Comment:
Hey, @os-lmo ...it has been a long time since I've done this PR. So, I need some time to switch and look at it again. That version was a temporary change in order to test. Of course the final version would follow the rules.